lib/models/GetData.rb

```diff
@@ -0,0 +1,10 @@
+class GetData
+  def load_data(path, data_name)
+    return unless File.exist?(path)
+
+    puts "Loading #{data_name} data from #{path}..."
+    loaded_data = JSON.parse(File.read(path))
+    puts "#{data_name.capitalize} data loaded successfully!"
+    loaded_data
+  end
+end
```

Comment:
- [ ] After exiting the application, the data was successfully saved :+1: . However, when attempting to load the saved data by running the application, I encountered the following error as shown below  👇 so make sure that you fix this error and ensure that the application still works as expected :pray: 
  
  ![image](https://github.com/PabloEGonz/school-library/assets/92303891/82640d8f-e8d5-4d10-831c-f0d4cdc876f7)